apps/src/lab2/levelEditors/validations/EditValidation.tsx

```diff
@@ -94,6 +94,22 @@ const EditValidation: React.FunctionComponent<EditValidationProps> = ({
           }}
         />
       </div>
+      <div className={moduleStyles.row}>
+        <label htmlFor="callout" className={moduleStyles.label}>
+          Callout (optional):
+        </label>
+        <textarea
```

Comment:
super nit: can this just be an `input` since it's a single line?

Reply:
Updated.